powerline/renderers/bar.py

```diff
@@ -0,0 +1,38 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+from powerline.renderer import Renderer
+from powerline.colorscheme import ATTR_BOLD, ATTR_ITALIC, ATTR_UNDERLINE
+
+
+class BarRenderer(Renderer):
+	'''BAR (Bar ain't recursive) rendere
+	(https://github.com/LemonBoy/bar)
```

Comment:
I guess it should either be removed completely or put into description (+1 empty line and it must contain some text in addition to the link). It is also highly suggested not to have link as text, but use link with some caption like just `LemonBoy/bar`.